msteams-platform/toolkit/build-a-basic-AI-chatbot-in-teams.md

```diff
@@ -118,9 +118,9 @@ Test Tool opens the bot in a webpage.
 | `env`        | Environment files.                                   |
 | `infra`      | Templates for provisioning Azure resources.          |
 | `src`        | The source code for the application.                 |
-|`teamsapp.yml`|This is the main Teams Toolkit project file. The project file defines two primary things:  Properties and configuration Stage definitions. |
-|`teamsapp.local.yml`|This overrides `teamsapp.yml` with actions that enable local execution and debugging.|
-|`teamsapp.testtool.yml`|This overrides `teamsapp.yml` with actions that enable local execution and debugging in Teams App Test Tool.|
+|`m365agents.yml`|This is the main Teams Toolkit project file. The project file defines two primary things:  Properties and configuration Stage definitions. |
+|`m365agents.local.yml`|This overrides `m365agents.yml` with actions that enable local execution and debugging.|
+|`teamsapp.testtool.yml`|This overrides `m365agents.yml` with actions that enable local execution and debugging in Teams App Test Tool.|
```

Comment:
Please cross verify if we need to update teamsapp.testtool.yml this file name as well.